doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -594,6 +594,159 @@ must be sent.
 *default location*, then the query will be declared as ambigous and an HTTP error response with code ``409``
 must be sent. 
 
+## Notifications formats
+
+Notifications include two fields: `subscriptionId` (which value is the one of the subscription triggering
+the notification) and `data` (which holds the actual data corresponding to entities and attributes).
+
+The `data` field can use different representation depending on two properties (especified as part of the
+corresponding subscription, see subscriptions operations section for details).
+
+* attrsFormat ("full", "keyValues", "values"; default is "full") to specify how the attributes are
```

Comment:
why don't we group all the options inside an 'options' dictionary or an options string like the one we have for queries.

```options: 'keyValues, {}'``` would mean key values represented as an object
```options: 'keyValues,[]'``` would mean key values represented as an array

another alternative could be:

````
"options": {
  "attrFormat": keyValues,
  "dataAs": "object"
}
````

options string is much more compact and could allow to us to be more orthogonal through the API